packages/commuter-directory-listing/src/DirectoryListing.js

```diff
@@ -26,10 +20,7 @@ const DirectoryListing = props => {
                     return (
                       <Table.Row key={index}>
                         <Table.Cell>
-                          <a
-                            href={`${row.path}`}
-                            onClick={handleClick(`${row.path}`)}
-                          >
+                          <a href={`${base}${row.path}`}>
```

Comment:
Less code while still handling relative paths with both /view and any base path.